### DISCUSSION TITLE
When to use a Service vs Driver?

### DISCUSSION THREAD

**Discussion by PeterQuinn925** [bellrichm/WeeWX-MQTTSubscribe#210] [General]
I didn't see this in the discussion or docs. I'm trying to decide if I should use a service or driver. I have a driver that I'm using for other data, can I have two drivers? Or, should I use a service to augment the existing driver with additional MQTT data from other sensors?

**Comment by bellrichm** [comment]
Yeah, its buried in the readme and could be easier to find. A WeeWX instance can only have one driver. So, if you a running instance that you want to augment, you will need to run as a service.
An alternative is to run multiple instances of WeeWX.
